ruby-generate-dockerfile/app/app_config.rb

```diff
@@ -179,7 +179,7 @@ def init_entrypoint
   def decorate_entrypoint entrypoint
     return JSON.generate entrypoint if entrypoint.is_a? Array
     return entrypoint if entrypoint.start_with? "exec "
-    return entrypoint if entrypoint =~ /;|&&|\|/
+    return entrypoint if entrypoint =~ /=|;|&&|\|/
```

Comment:
Yes there could be. This is not and never will be an exact check. It's meant to be conservative, to prevent prepending "exec" in _all_ cases where doing so would break the entrypoint, acknowledging that it is okay but not ideal to have false positives. That said, a more stringent check would still be useful here. I'll take a stab at it.

Reply:
Tightened the check.